packages/core/src/external/aws/lambda-logic/document-uploader.ts

```diff
@@ -80,6 +80,11 @@ export async function documentUploaderHandler(
     const docRef = await forwardCallToServer(cxId, apiServerURL, fileData);
     const stringSize = size ? size.toString() : "";
     const hash = eTag ? eTag : "";
+    if (!contentType) {
+      const message = "Failed to get the mime type of the uploaded file";
+      console.log(`${message}: ${contentType}`);
+      throw new MetriportError(message, null, { sourceKey, destinationKey });
+    }
```

Comment:
Thanks, but just to confirm, did you look at the downstream logic to make sure the content/mime type is required? We wouldn't want to fail to process a document b/c of missing mime type if thats not required - i.e., could make it optional on the resulting XML.